ansible/deploy.py

```diff
@@ -30,8 +31,22 @@
 @click.option("--tags", "--roles", help="Roles to execute")
 @click.option("--check", is_flag=True, default=False, help="Perform a dry run")
 @click.option("--force", is_flag=True, default=False, help="Override checks")
+@click.option(
+    "--from",
+    "from_playbook",
+    help="Determine from which playbook to start the deploy from",
+)
+@click.option(
+    "--until", "until_playbook", help="Determine until which playbook to run the deploy"
```

Comment:
there is only one playbook but multiple roles. The text should be `until which role` (same for the `--from` option)